include/flexflow/graph.h

```diff
@@ -84,11 +84,11 @@ struct NodeCompare {
 };
 
 struct GraphOptimalViewSerialized {
-  //#ifdef LEGION_MAX_RETURN_SIZE
-  //  static const size_t buffer_size = LEGION_MAX_RETURN_SIZE - 8;
-  //#else
+#ifdef LEGION_MAX_RETURN_SIZE
```

Comment:
What's the reason for this change?

Reply:
There is a static assert in legion that breaks if we don't cap the buffer

Reply:
I guess the question more was why was this change not necessary before?

Reply:
Ah gotcha yeah don't know. 

https://github.com/flexflow/FlexFlow/commit/3249a2c5b4ba65b37dc621bd3a3906e2bde0c509

@jiazhihao can you provide some color on why it was commented out in this commit?

Reply:
This change relates to supporting serialization for very large models. Actually, I don't like the changes I made --- it's kind of a hack. `LEGION_MAX_RETURN_SIZE` specifies the maximum size of a `Future` returned by a Legion task. We want to specify it in `cmake` and the specified value should be passed to both Legion and FlexFlow compilation.

Reply:
@jiazhihao Maybe worth making a followup issue for that?

Reply:
#417 tracks this issue.

Reply:
perf!